### ISSUE TITLE
Unprivileged user containers not starting

### ISSUE THREAD

**Issue by laerus** [lxc/lxc#2575]
(LOOK COMMENT BELLOW)
# Required information

 * Distribution:
Manjaro

 * The output of
   * `lxc-start --version`

`3.0.2`
   * `lxc-checkconfig`

```
--- Namespaces ---
Namespaces: enabled
Utsname namespace: enabled
Ipc namespace: enabled
Pid namespace: enabled
User namespace: enabled
Network namespace: enabled

--- Control groups ---
Cgroups: enabled

Cgroup v1 mount points: 
/sys/fs/cgroup/systemd
/sys/fs/cgroup/net_cls,net_prio
/sys/fs/cgroup/blkio
/sys/fs/cgroup/rdma
/sys/fs/cgroup/cpuset
/sys/fs/cgroup/devices
/sys/fs/cgroup/freezer
/sys/fs/cgroup/cpu,cpuacct
/sys/fs/cgroup/hugetlb
/sys/fs/cgroup/pids
/sys/fs/cgroup/memory
/sys/fs/cgroup/perf_event

Cgroup v2 mount points: 
/sys/fs/cgroup/unified

Cgroup v1 clone_children flag: enabled
Cgroup device: enabled
Cgroup sched: enabled
Cgroup cpu account: enabled
Cgroup memory controller: enabled
Cgroup cpuset: enabled

--- Misc ---
Veth pair device: enabled, not loaded
Macvlan: enabled, not loaded
Vlan: enabled, loaded
Bridges: enabled, loaded
Advanced netfilter: enabled, not loaded
CONFIG_NF_NAT_IPV4: enabled, loaded
CONFIG_NF_NAT_IPV6: enabled, not loaded
CONFIG_IP_NF_TARGET_MASQUERADE: enabled, loaded
CONFIG_IP6_NF_TARGET_MASQUERADE: enabled, not loaded
CONFIG_NETFILTER_XT_TARGET_CHECKSUM: enabled, loaded
CONFIG_NETFILTER_XT_MATCH_COMMENT: enabled, not loaded
FUSE (for use with lxcfs): enabled, loaded

--- Checkpoint/Restore ---
checkpoint restore: enabled
CONFIG_FHANDLE: enabled
CONFIG_EVENTFD: enabled
CONFIG_EPOLL: enabled
CONFIG_UNIX_DIAG: enabled
CONFIG_INET_DIAG: enabled
CONFIG_PACKET_DIAG: enabled
CONFIG_NETLINK_DIAG: enabled
File capabilities: 

Note : Before booting a new kernel, you can check its configuration
usage : CONFIG=/path/to/config /usr/bin/lxc-checkconfig
```
   * `uname -a`
```Linux 4.17.18-1-MANJARO #1 SMP PREEMPT Wed Aug 22 21:51:13 UTC 2018 x86_64 GNU/Linux```
   * `cat /proc/self/cgroup`
```
12:perf_event:/
11:memory:/user.slice/user-1000.slice/session-c2.scope
10:pids:/user.slice/user-1000.slice/session-c2.scope
9:hugetlb:/
8:cpu,cpuacct:/user.slice
7:freezer:/
6:devices:/user.slice
5:cpuset:/
4:rdma:/
3:blkio:/user.slice
2:net_cls,net_prio:/
1:name=systemd:/user.slice/user-1000.slice/session-c2.scope
0::/user.slice/user-1000.slice/session-c2.scope
```

   * `cat /proc/1/mounts`
```
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
sys /sys sysfs rw,nosuid,nodev,noexec,relatime 0 0
dev /dev devtmpfs rw,nosuid,relatime,size=4029336k,nr_inodes=1007334,mode=755 0 0
run /run tmpfs rw,nosuid,nodev,relatime,mode=755 0 0
/dev/sdb1 / ext4 rw,noatime,discard 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,mode=755 0 0
cgroup2 /sys/fs/cgroup/unified cgroup2 rw,nosuid,nodev,noexec,relatime,nsdelegate 0 0
cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,name=systemd 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
bpf /sys/fs/bpf bpf rw,nosuid,nodev,noexec,relatime,mode=700 0 0
cgroup /sys/fs/cgroup/net_cls,net_prio cgroup rw,nosuid,nodev,noexec,relatime,net_cls,net_prio 0 0
cgroup /sys/fs/cgroup/blkio cgroup rw,nosuid,nodev,noexec,relatime,blkio 0 0
cgroup /sys/fs/cgroup/rdma cgroup rw,nosuid,nodev,noexec,relatime,rdma 0 0
cgroup /sys/fs/cgroup/cpuset cgroup rw,nosuid,nodev,noexec,relatime,cpuset 0 0
cgroup /sys/fs/cgroup/devices cgroup rw,nosuid,nodev,noexec,relatime,devices 0 0
cgroup /sys/fs/cgroup/freezer cgroup rw,nosuid,nodev,noexec,relatime,freezer 0 0
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpu,cpuacct 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup rw,nosuid,nodev,noexec,relatime,hugetlb 0 0
cgroup /sys/fs/cgroup/pids cgroup rw,nosuid,nodev,noexec,relatime,pids 0 0
cgroup /sys/fs/cgroup/memory cgroup rw,nosuid,nodev,noexec,relatime,memory 0 0
cgroup /sys/fs/cgroup/perf_event cgroup rw,nosuid,nodev,noexec,relatime,perf_event 0 0
hugetlbfs /dev/hugepages hugetlbfs rw,relatime,pagesize=2M 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
systemd-1 /proc/sys/fs/binfmt_misc autofs rw,relatime,fd=41,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=11651 0 0
debugfs /sys/kernel/debug debugfs rw,relatime 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,relatime 0 0
configfs /sys/kernel/config configfs rw,relatime 0 0
tmpfs /tmp tmpfs rw,noatime 0 0
/etc/autofs/auto.misc /misc autofs rw,relatime,fd=6,pgrp=482,timeout=300,minproto=5,maxproto=5,indirect,pipe_ino=18564 0 0
-hosts /net autofs rw,relatime,fd=12,pgrp=482,timeout=300,minproto=5,maxproto=5,indirect,pipe_ino=15707 0 0
/etc/autofs/auto.modulus_nas /mnt/modulus_nas autofs rw,relatime,fd=18,pgrp=482,timeout=600,minproto=5,maxproto=5,indirect,pipe_ino=19478 0 0
tmpfs /run/user/1000 tmpfs rw,nosuid,nodev,relatime,size=807184k,mode=700,uid=1000,gid=1000 0 0
fusectl /sys/fs/fuse/connections fusectl rw,relatime 0 0
gvfsd-fuse /run/user/1000/gvfs fuse.gvfsd-fuse rw,nosuid,nodev,relatime,user_id=1000,group_id=1000 0 0
```

# Issue description
Cannot start user unprivileged containers

```
lxc-start test1 20180828114941.265 INFO     confile - confile.c:set_config_idmaps:1542 - Read uid map: type u nsid 0 hostid 1258512 range 65536
lxc-start test1 20180828114941.265 INFO     confile - confile.c:set_config_idmaps:1542 - Read uid map: type g nsid 0 hostid 1258512 range 65536
lxc-start test1 20180828114941.266 INFO     lxccontainer - lxccontainer.c:do_lxcapi_start:959 - Set process title to [lxc monitor] /home/kon/.local/share/lxc test1
lxc-start test1 20180828114941.267 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 23
lxc-start test1 20180828114941.267 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 30
lxc-start test1 20180828114941.267 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 31
lxc-start test1 20180828114941.267 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 32
lxc-start test1 20180828114941.267 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 34
lxc-start test1 20180828114941.267 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 35
lxc-start test1 20180828114941.267 INFO     lsm - lsm/lsm.c:lsm_init:47 - LSM security driver nop
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "reject_force_umount  # comment this to allow umount -f;  not recommended"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for reject_force_umount action 0(kill)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for reject_force_umount action 0(kill)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for reject_force_umount action 0(kill)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for reject_force_umount action 0(kill)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "[all]"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "kexec_load errno 1"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for kexec_load action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for kexec_load action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for kexec_load action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for kexec_load action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "open_by_handle_at errno 1"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for open_by_handle_at action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for open_by_handle_at action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for open_by_handle_at action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for open_by_handle_at action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "init_module errno 1"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for init_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for init_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for init_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for init_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "finit_module errno 1"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for finit_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for finit_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for finit_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for finit_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "delete_module errno 1"
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for delete_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for delete_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for delete_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for delete_module action 327681(errno)
lxc-start test1 20180828114941.267 INFO     seccomp - seccomp.c:parse_config_v2:967 - Merging compat seccomp contexts into main context
lxc-start test1 20180828114941.267 DEBUG    terminal - terminal.c:lxc_terminal_peer_default:704 - No such device - The process does not have a controlling terminal
lxc-start test1 20180828114941.267 DEBUG    conf - conf.c:chown_mapped_root:3166 - trying to chown "/dev/pts/3" to 1000
lxc-start test1 20180828114941.329 ERROR    cgfsng - cgroups/cgfsng.c:all_controllers_found:701 - No freezer controller mountpoint found
lxc-start test1 20180828114941.329 ERROR    cgroup - cgroups/cgroup.c:cgroup_init:44 - Failed to initialize cgroup driver
lxc-start test1 20180828114941.329 ERROR    start - start.c:lxc_init:861 - Failed to initialize cgroup driver
lxc-start test1 20180828114941.329 ERROR    start - start.c:__lxc_start:1876 - Failed to initialize container "test1"
lxc-start test1 20180828114941.330 DEBUG    lxccontainer - lxccontainer.c:wait_on_daemonized_start:828 - First child 4596 exited
lxc-start test1 20180828114941.330 ERROR    lxccontainer - lxccontainer.c:wait_on_daemonized_start:831 - No such file or directory - Failed to receive the container state
lxc-start test1 20180828114941.330 ERROR    lxc_start - tools/lxc_start.c:main:330 - The container failed to start
lxc-start test1 20180828114941.330 ERROR    lxc_start - tools/lxc_start.c:main:333 - To get more details, run the container in foreground mode
lxc-start test1 20180828114941.330 ERROR    lxc_start - tools/lxc_start.c:main:336 - Additional information can be obtained by setting the --logfile and --logpriority options

```

**Comment by laerus** [comment]
apparently this is stiill an issue with:
kernel 4.18.4
lxc 3.0.2

```
lxc-start ldap-server 20180829092902.988 INFO     confile - confile.c:set_config_idmaps:1542 - Read uid map: type u nsid 0 hostid 1258512 range 65536
lxc-start ldap-server 20180829092902.988 INFO     confile - confile.c:set_config_idmaps:1542 - Read uid map: type g nsid 0 hostid 1258512 range 65536
lxc-start ldap-server 20180829092902.988 INFO     lxccontainer - lxccontainer.c:do_lxcapi_start:959 - Set process title to [lxc monitor] /home/kon/.local/share/lxc ldap-server
lxc-start ldap-server 20180829092902.989 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 21
lxc-start ldap-server 20180829092902.989 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 23
lxc-start ldap-server 20180829092902.989 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 32
lxc-start ldap-server 20180829092902.989 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 33
lxc-start ldap-server 20180829092902.989 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 34
lxc-start ldap-server 20180829092902.989 INFO     start - start.c:lxc_check_inherited:286 - Closed inherited fd 36
lxc-start ldap-server 20180829092902.989 INFO     lsm - lsm/lsm.c:lsm_init:47 - LSM security driver nop
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "reject_force_umount  # comment this to allow umount -f;  not recommended"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for reject_force_umount action 0(kill)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for reject_force_umount action 0(kill)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for reject_force_umount action 0(kill)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:do_resolve_add_rule:503 - Set seccomp rule to reject force umounts
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for reject_force_umount action 0(kill)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "[all]"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "kexec_load errno 1"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for kexec_load action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for kexec_load action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for kexec_load action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for kexec_load action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "open_by_handle_at errno 1"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for open_by_handle_at action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for open_by_handle_at action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for open_by_handle_at action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for open_by_handle_at action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "init_module errno 1"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for init_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for init_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for init_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for init_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "finit_module errno 1"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for finit_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for finit_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for finit_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for finit_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:757 - Processing "delete_module errno 1"
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:934 - Added native rule for arch 0 for delete_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:943 - Added compat rule for arch 1073741827 for delete_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:953 - Added compat rule for arch 1073741886 for delete_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:963 - Added native rule for arch -1073741762 for delete_module action 327681(errno)
lxc-start ldap-server 20180829092902.989 INFO     seccomp - seccomp.c:parse_config_v2:967 - Merging compat seccomp contexts into main context
lxc-start ldap-server 20180829092902.989 DEBUG    terminal - terminal.c:lxc_terminal_peer_default:704 - No such device - The process does not have a controlling terminal
lxc-start ldap-server 20180829092902.989 DEBUG    conf - conf.c:chown_mapped_root:3166 - trying to chown "/dev/pts/2" to 1000
lxc-start ldap-server 20180829092903.310 ERROR    cgfsng - cgroups/cgfsng.c:all_controllers_found:701 - No freezer controller mountpoint found
lxc-start ldap-server 20180829092903.311 ERROR    cgroup - cgroups/cgroup.c:cgroup_init:44 - Failed to initialize cgroup driver
lxc-start ldap-server 20180829092903.311 ERROR    start - start.c:lxc_init:861 - Failed to initialize cgroup driver
lxc-start ldap-server 20180829092903.311 ERROR    start - start.c:__lxc_start:1876 - Failed to initialize container "ldap-server"
lxc-start ldap-server 20180829092903.314 DEBUG    lxccontainer - lxccontainer.c:wait_on_daemonized_start:828 - First child 2695 exited
lxc-start ldap-server 20180829092903.315 ERROR    lxccontainer - lxccontainer.c:wait_on_daemonized_start:831 - No such file or directory - Failed to receive the container state
lxc-start ldap-server 20180829092903.315 ERROR    lxc_start - tools/lxc_start.c:main:330 - The container failed to start
lxc-start ldap-server 20180829092903.315 ERROR    lxc_start - tools/lxc_start.c:main:333 - To get more details, run the container in foreground mode
lxc-start ldap-server 20180829092903.315 ERROR    lxc_start - tools/lxc_start.c:main:336 - Additional information can be obtained by setting the --logfile and --logpriority options
```

**Comment by laerus** [comment]
is `pam_cgfs` part of `liblxc` ? Archlinux doesn't seem to have a different package for `liblxc` like Debian, i will have to ask downstream.

**Comment by brauner** [comment]
Starting with `LXC 3.0` `pam_cgfs` is shipped with `liblxc`. Prior to that it was part of `LXCFS`. Afaict, ArchLinux has never made use of `pam_cgfs`.

**Comment by laerus** [comment]
i'm actually using Manjaro that compiles the kernel with user namespaces. (not sure if this is relevant for pam_cgfs)

i see this issue is a duplicate and you're trying to solve it already, closing this, thx for the info!

**Comment by laerus** [comment]
apparently even on Archlinux this line is needed on pam `session    optional   pam_cgfs.so -c freezer,memory,name=systemd,unified` else i get the same error about no freezer mount.

**Comment by brauner** [comment]
But then it works?

**Comment by laerus** [comment]
no sadly they don't seem to have pulled the pam fixes for 3.0.2 yet, 
https://bugs.archlinux.org/task/59689?project=5&string=lxc 

thank you for your time @brauner !
